project/ProjectPlugin.scala

```diff
@@ -21,14 +19,13 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val V = new {
       val avro4s: String             = "1.8.3"
+      val catsEffect: String         = "0.9"
       val frees: String              = "0.7.0"
       val fs2ReactiveStreams: String = "0.5.1"
-      val grpc: String               = "1.9.1"
-      val monix: String              = "3.0.0-M3"
+      val grpc: String               = "1.10.0"
```

Comment:
Note to remember, next release should be a minor version ;)

Reply:
Ah, because we bump a minor version in a dependency I guess. Good to know

Reply:
Yep, potentially 1.10.0 is bringing breaking changes we need to keep in mind.